verify/cert.go

```diff
@@ -22,7 +22,15 @@ import (
 	"golang.org/x/exp/slices"
 )
 
-// CertVerifier verifies the DA certificate against on-chain EigenDA contracts
+type certVerifier interface {
+	verifyBatchConfirmedOnChain(ctx context.Context, batchID uint32, batchMetadata *disperser.BatchMetadata) error
+	verifyMerkleProof(inclusionProof []byte, root []byte, index uint32, blobHeader BlobHeader) error
+	// getters for the cached quorum parameters
+	quorumNumbersRequired() []uint8
+	quorumAdversaryThresholdPercentages(quorum uint8) (percentage uint8, ok bool)
+}
+
+// certVerification verifies the DA certificate against on-chain EigenDA contracts
```

Comment:
Was it intentional to change this to say `certVerification` here, instead of saying `CertVerifier`?

Reply:
674a41cc